File: foundation/foundation 2 month/p_exam/10-10-24/v2/ex_1.c
#include <stdio.h>

void check(double a,double b)
{
	double  c=a+b;
	printf("%.2lf",c);
}
int main()
{
	double x,y;
	printf("X ni kiriting: ");
	scanf("%lf",&x);
	printf("Y ni kiriting: ");
	scanf("%lf",&y);
	check(x,y);
return 0;
}

File: foundation/foundation 3 month/lesson10/homework/ex_1.py
import random
def check(x):
	with open(x,'wt') as w:
		m=int(input("elementlar soni kiriting: "))
		for x in range(m):
			w.write(str(random.randint(10,99))+',')

		print("Fayl kiriting")


def chec(x):
	with open(x,'rt') as r:
		ms=list()
		d=r.read().split(',')
		d.pop(-1)
		print(d)
		for x in d:
			if int(x)%2==0:
				ms.append(x)
		ms=ms[::-1]
		print(ms)


k=input("Fayl nomini kiritng: ")
check(k)
chec(k)
